site/src/pages/index.js

```diff
@@ -1,79 +1,134 @@
 import { Main } from '../sitecomponents/App.style';
-import Footer from "../sitecomponents/Footer/index";
-import { ThemeProvider } from "styled-components";
-import { darkTheme, GlobalStyle, lightTheme } from '../sitecomponents/index.style.js';
-import { useDarkMode } from "../sitecomponents/useDarkMode";
-import Navigation from "../sitecomponents/Navigation/index";
-import Faq from "../sitecomponents/Faq/index";
-import React, { useState } from "react";
-import BadgeGrid from "../sitecomponents/BadgeGrid/index";
-import recognitionLogo from "../assets/images/recognition-program.png";
-import recognitionBanner from "../assets/images/recognition-banner.png";
-import githubGopi from "../assets/images/github-badges.png";
-import githubKaran from "../assets/images/github-badges-karan.png";
-import '../fonts.css'
+import Footer from '../sitecomponents/Footer/index';
+import { ThemeProvider } from 'styled-components';
+import {
+  darkTheme,
+  GlobalStyle,
+  lightTheme,
+} from '../sitecomponents/index.style.js';
+import { useDarkMode } from '../sitecomponents/useDarkMode';
+import Navigation from '../sitecomponents/Navigation/index';
+import Faq from '../sitecomponents/Faq/index';
+import React, { useState } from 'react';
+import BadgeGrid from '../sitecomponents/BadgeGrid/index';
+import recognitionLogo from '../assets/images/recognition-program.png';
+import recognitionBanner from '../assets/images/recognition-banner.png';
+import githubGopi from '../assets/images/github-badges.png';
```

Comment:
what are these badges?

Reply:
Not sure I added those, but I think its should be one of the badges on the homepage

Reply:
It’s Gopi’s screenshot of his GitHub profile with badges on it. It’s a poor quality screenshot and not well presented on the site. Please remove it.

Reply:
Done